### ISSUE TITLE
Suggestion: more ghost blocks to fill in certain shapes.

### ISSUE THREAD

**Issue by llloyd4** [Ipsis/BuildersGuides#16]
Hello! I was making an under water dome and I noticed the guide ghost blocks didn't really fill in 100%, I had to do a lot of guessing as to where I needed to fill in and should I put a block here or there to make the dome water tight.  Maybe make this more like the OpenBlock version of the Build Guide? 

[URL=http://s15.photobucket.com/user/llloyd4/media/2017-05-28_23.10.16_zps0grwp0vr.png.html][IMG]http://i15.photobucket.com/albums/a395/llloyd4/2017-05-28_23.10.16_zps0grwp0vr.png~original[/IMG][/URL]

**Comment by Ipsis** [comment]
Simply put - the OpenBlocks maths algorithms are far superior than the ones that I implemented. :(

Builder's Guides was always supposed to complement OpenBlocks, which is why I only added minimal shape support while waiting for the OpenBlocks port.

I may eventually get around to updating the shapes, but I struggled to find drawing algorithms that I was happy to use.

**Comment by llloyd4** [comment]
Yeah, I know, there are a bunch of mods still, to this day, are missing on 1.10.2, some skipped to 1.11 and avoided 1.10. >.< Now with 1.12 being officially released soon with yet another massive update, oy. >.<
